Makefile

```diff
@@ -161,6 +161,10 @@ CREATE_KIND_CLUSTER ?= true
 .PHONY: test-e2e
 test-e2e: kustomize manifests generate ginkgo run-test-e2e-$(E2E_KIND_VERSION:kindest/node:v%=%)
 
+.PHONY: test-e2e-experimental
```

Comment:
let's remove any trace of the experimental folder in the main Makefile

Reply:
Will do

Reply:
How would you like to handle the common bin/ depedencies like `kind`?

Reply:
Unless you have another preference, I will reference them by relative path and specify in the test steps to execute the primary Makefile first to download those tools.

Reply:
Done